### DISCUSSION TITLE
MC server via docker on macOS

### DISCUSSION THREAD

**Discussion by ghost** [Ylianst/MeshCentral#4485] [Q&A]
Is there a complete handholding idiots guide to installing and configuring MC in docker on macOS. 

I'm looking to replace my current rmm and like what I see but I'm using the public server and would rather run my own server.

**Comment by Ylianst** [comment]
I am not an expert on Docker, but you can [search for `MeshCentral` on DockerHub](https://hub.docker.com/search?q=meshcentral), there are a few images there. A lot of people setup MeshCentral using Docker, but oddly, I have never done it. I keep doing it the old fashion way.
